src/containers/AboutUsPage/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const AboutPage = () => {
+  const { t } = useTranslation();
+
+  // ToDo(jennethydyrova): Add team members' info and create card for each member
```

Comment:
I remember louis saying we should not add comments like this to Jamil but I am not sure :D

Reply:
on contrary, Louis asked to add comments when work is not finished

Reply:
Ok my bad I guess I remembered wrong :D

Reply:
no worries!